3rd-Party.license

```diff
@@ -21,6 +21,10 @@ javax.inject        - The javax.inject API (https://www.apache.org/licenses/LICE
 License:              Apache 2.0
 License File:         javax.inject.license
 
+javax.enterprise.cdi-api - Contexts and Dependency Injection for Java (https://github.com/eclipse-ee4j/cdi/blob/master/LICENSE.txt)
+License:              Apache 2.0
+License File          javax.enterprise.cdi-api
+
```

Comment:
copy to
https://github.com/ONLYOFFICE/onlyoffice-confluence/blob/master/licenses/3rd-Party.license